languages.toml

```diff
@@ -766,6 +767,23 @@ indent = { tab-width = 2, unit = "  " }
 name = "typescript"
 source = { git = "https://github.com/tree-sitter/tree-sitter-typescript", rev = "b1bf4825d9eaa0f3bdeb1e52f099533328acfbdf", subpath = "typescript" }
 
+[[language]]
+name = "typespec"
+scope = "source.typespec"
+injection-regex = "(tsp|typespec)"
+language-id = "typespec"
+file-types = ["tsp"]
+roots = ["tspconfig.yaml"]
+auto-format = true
+comment-token = "//"
+block-comment-tokens = { start = "/*", end = "*/" }
+language-servers = ["typespec"]
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "typespec"
+source = { git = "https://github.com/happenslol/tree-sitter-typespec", rev = "af7a97eea5d4c62473b29655a238d4f4e055798b" }
```

Comment:
This grammar repository doesn't currently have a license. Downstream packagers need these to have licenses to distribute them: could you propose a license and update the revision to one that includes the license when it's added?

Reply:
Opened happenslol/tree-sitter-typespec#5 for the author to add one.